README.md

```diff
@@ -83,30 +126,52 @@ git submodule update --init --recursive
 set CMAKE_ARGS=-DONNX_USE_LITE_PROTO=ON
 pip install -e .
 ```
+### Linux
+
+First, you need to install protobuf. 
+
+Ubuntu users: the quickest way to install protobuf is to run
 
-### Building Protobuf from source
-* **Windows**
+```bash
+apt-get install python3-pip python3-dev libprotobuf-dev protobuf-compiler
+export CMAKE_ARGS="-DONNX_USE_PROTOBUF_SHARED_LIBS=ON"
+```
+Otherwise, if you want install it from source you can use the following commands:
+
+Debian/Ubuntu:
 ```
 git clone https://github.com/protocolbuffers/protobuf.git
 cd protobuf
 git checkout v3.16.0
-cd cmake
-cmake -G "Visual Studio 16 2019" -A x64 -DCMAKE_INSTALL_PREFIX=<protobug_install_dir> -Dprotobuf_MSVC_STATIC_RUNTIME=ON -Dprotobuf_BUILD_SHARED_LIBS=OFF -Dprotobuf_BUILD_TESTS=OFF -Dprotobuf_BUILD_EXAMPLES=OFF .
-msbuild protobuf.sln /m /p:Configuration=Release
-msbuild INSTALL.vcxproj /p:Configuration=Release
+git submodule update --init --recursive
+mkdir build_source && cd build_source
+cmake ../cmake -Dprotobuf_BUILD_SHARED_LIBS=OFF -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_INSTALL_SYSCONFDIR=/etc -DCMAKE_POSITION_INDEPENDENT_CODE=ON -Dprotobuf_BUILD_TESTS=OFF -DCMAKE_BUILD_TYPE=Release
+make -j$(nproc)
+make install
 ```
 
-* **Linux**
+CentOS/RHEL/Fedora:
 ```
 git clone https://github.com/protocolbuffers/protobuf.git
 cd protobuf
 git checkout v3.16.0
 git submodule update --init --recursive
 mkdir build_source && cd build_source
-cmake ../cmake -Dprotobuf_BUILD_SHARED_LIBS=OFF -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_INSTALL_SYSCONFDIR=/etc -DCMAKE_POSITION_INDEPENDENT_CODE=ON -Dprotobuf_BUILD_TESTS=OFF -DCMAKE_BUILD_TYPE=Release
+cmake ../cmake  -DCMAKE_INSTALL_LIBDIR=lib64 -Dprotobuf_BUILD_SHARED_LIBS=OFF -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_INSTALL_SYSCONFDIR=/etc -DCMAKE_POSITION_INDEPENDENT_CODE=ON -Dprotobuf_BUILD_TESTS=OFF -DCMAKE_BUILD_TYPE=Release
```

Comment:
when -DONNX_USE_PROTOBUF_SHARED_LIBS=ON, -Dprotobuf_BUILD_SHARED_LIBS should also be ON